packages/web-features-figuring/src/baseline/index.ts

```diff
@@ -9,6 +9,11 @@ import { support } from "./support";
 import { Browser } from "../browser-compat-data/browser";
 import { Compat, defaultCompat } from "../browser-compat-data/compat";
 
+export const BASELINE_HIGH_DURATION = Temporal.Duration.from({ months: 30 });
```

Comment:
We have this constant in the top-level index.ts too. Can it be shared?

Reply:
Yes, done with a01258d. Though I imagine needing to do so will be temporary, until we can call `computeBaseline` from `index.ts` instead.